docs/reference/tctl/workflow/show.md

```diff
@@ -0,0 +1,101 @@
+---
+id: show
+title: tctl workflow show
+description: How to show Workflow History using tctl.
+tags:
+  - reference
+  - tctl
+---
+
+<!-- prettier-ignore -->
+import * as WhatIsAWorkflowExecution from '../../../content/what-is-a-workflow-execution.md'
+import * as WhatIsAWorkflowId from '../../../content/what-is-a-workflow-id.md'
+import * as WhatIsARunId from '../../../content/what-is-a-run-id.md'
+
+The `tctl workflow show` command shows History for the specified <preview page={WhatIsAWorkflowExecution}>Workflow Execution</preview>.
+
+**Syntax** `tctl workflow show <command options> <arguments...>`
```

Comment:
Might look cleaner to have this in its own code block?

Reply:
Because it's a syntax line, it's not really something that users would want to copy, I think. However, removing the "Syntax" run-in heading might accomplish the same thing. I'm not sure that the heading is really needed.